pkg/proxy/nftables/proxier_test.go

```diff
@@ -875,7 +883,7 @@ func TestNodePorts(t *testing.T) {
 			family:            v1.IPv4Protocol,
```

Comment:
github won't let me comment outside the diff, but... I'm noticing that this test function mentions `--localhost-nodeports` in the comment at the top of the function, which is wrong, so can you remove that.

Also, the `expectFirewall` field is still here, but it's not actually used by the test, so can you remove that too?